docs/tractus-x-release.md

```diff
@@ -0,0 +1,21 @@
+# Tractus-X Release
+
+## Artifacts
+
+- Umbrella Helm Chart(s)
+- Changelog (overarching feat)
```

Comment:
I'd put the changelog in first position as it's the already established artifact